app/src/main/java/org/oppia/android/app/player/exploration/BottomSheetOptionsMenu.kt

```diff
@@ -0,0 +1,40 @@
+package org.oppia.android.app.player.exploration
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import org.oppia.android.R
+import org.oppia.android.databinding.FragmentBottomSheetBinding
+
+/** Bottom sheet fragment for displaying options menu */
+class BottomSheetOptionsMenu(val internalProfileId: Int) : BottomSheetDialogFragment() {
```

Comment:
Does this parameter need to be passed in through arguments, instead, since Android can recreate the fragment itself?